feat(docs): Explanation about time sync (#1627)
canonical/ubuntu-pro-for-wsl: docs/explanation/index.md

binfmt
```

## Network time synchronization for Ubuntu on WSL

> Learn more about how Ubuntu on WSL synchronizes time and what to do if it doesn't meet your needs.

Reviewer: ```suggestion
Learn more about how Ubuntu on WSL synchronizes time and your configuration options for time synchronization.
```
